nrf52-hal-common/src/lib.rs

```diff
@@ -41,18 +41,49 @@ pub mod target_constants {
     // NRF52832 8 bits1..0xFF
     pub const EASY_DMA_SIZE: usize = 255;
     // Easy DMA can only read from data ram
-    pub const SRAM_LOWER:usize = 0x2000_0000;
-    pub const SRAM_UPPER:usize = 0x3000_0000;
-    pub const FORCE_COPY_BUFFER_SIZE:usize = 255;
+    pub const SRAM_LOWER: usize = 0x2000_0000;
+    pub const SRAM_UPPER: usize = 0x3000_0000;
+    pub const FORCE_COPY_BUFFER_SIZE: usize = 255;
```

Comment:
Well, but for the nRF52840 this is not the `EASY_DMA_SIZE` ;) Maybe it would be feasible to use it there too but on the nRF52840 this would occupy 16kb of ram ;)

Reply:
Yeah, I did this for @Yatekii's reason, I don't think it's reasonable to eat 16k of ram here. Though I do agree I should document that 255 bytes is a semi-arbitrary value that is <= the lowest EASY_DMA_SIZE (== the 810/832 len)

Reply:
@Yatekii  yes, I was only thinking about the 810 and 832 here, as the comment was about code inside the conditional block. @jamesmunns yes a single comment here for the future maintainer of this code.